reference-data-proxy/src/cronJobs/eStoreTermStoreAndBuyerFolder.cron.ts

```diff
@@ -48,8 +50,8 @@ export const eStoreTermStoreAndBuyerFolder = async (eStoreData: any) => {
     { returnDocument: 'after' },
   );
 
-  // ensure that there is only 1 retry for buyer folder creation
-  if (response?.value?.buyerFolderRetries === 1) {
+  // ensure that there is less than 3 retries for buyer folder creation
+  if (response?.value?.buyerFolderRetries <= 3) {
```

Comment:
for the future, wondering if its best to put 3 and 99 in constants so easier to change?